builtin/audit/file/backend.go

```diff
@@ -258,6 +258,24 @@ func (b *Backend) LogResponse(ctx context.Context, in *logical.LogInput) error {
 	return b.log(ctx, buf, writer)
 }
 
+func (b *Backend) LogTestMessage(ctx context.Context, in *logical.LogInput, config map[string]string) error {
+	var writer io.Writer
+	switch b.path {
+	case "stdout":
+		writer = os.Stdout
+	case "discard":
+		return nil
+	}
```

Comment:
should there be a default case? otherwise i'd think `writer` could be nil?

Reply:
writer can be nil, but confusingly for this backend the write function uses open() when it is